APIComparer.Backend.Tests/APIComparer.Backend.Tests.csproj

```diff
@@ -69,6 +85,9 @@
   <ItemGroup>
     <Folder Include="Properties\" />
   </ItemGroup>
+  <ItemGroup>
+    <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
```

Comment:
I've eventually dropped it from this PR. This was adding far too much noise to the diff.